backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java

```diff
@@ -308,6 +311,19 @@ public void onEvent(BacktraceResult backtraceResult) {
         }
     }
 
+    @Test
+    public void verifyBreadcrumbCallbackInvocation() {
+        backtraceClient.enableBreadcrumbs(context);
+        Breadcrumbs breadcrumbs = backtraceClient.database.getBreadcrumbs();
+
+        breadcrumbs.setOnSuccessfulBreadcrumbAddEventListener(breadcrumbId -> {
+            assertEquals(breadcrumbs.getCurrentBreadcrumbId(), breadcrumbId);
+            return;
+        });
+
+        breadcrumbs.addBreadcrumb("test");
```

Comment:
are you sure that this assert is triggered?

Reply:
yeah Im sure - it's a sync call. I also verified it in the debugger